scripts/bin/update-packages

```diff
@@ -303,7 +421,7 @@ _update_dependencies() {
 			termux_error_exit "ERROR: Obtaining update order failed for $(basename "${pkg_dir}")"
 		fi
 		_should_update "${dep_dir}"  && ! _check_updated "${dep_dir}" && _run_update "${dep_dir}"
-	done <<<"$("${TERMUX_SCRIPTDIR}"/scripts/buildorder.py "${pkg_dir}" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")"
+	done < <("${TERMUX_SCRIPTDIR}"/scripts/buildorder.py "${pkg_dir}" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")
```

Comment:
That is a small change which is mostly cosmetic, not directly or indirectly related to the problem this PR is going to solve. It can be tested and merged separately.